README.md

```diff
@@ -903,7 +903,7 @@ personalisation={
 
 ## Get received text messages with pagination
 
-This will return one page of text messages (250) per call. Use the `get_received_texts_iterator` to retrieve all received texts unpaginated. 
+This will return one page of the latest received text messages (250) per call. Use the `get_received_texts_iterator` to retrieve all received texts unpaginated. 
```

Comment:
Would be clearer as a separate sentence that says what the order is, eg ‘Results are ordered by most recent first’.

Reply:
ok